lib/theme/dimens.dart

```diff
@@ -0,0 +1,11 @@
+const double space10 = 10.0;
```

Comment:
@doannimble Initially, I named these constants like this: `spaceSmall`, `spaceMedium`, `spaceLarge`, .etc. Even that, when we use the constant, we always need to check the value from my experience. So having the constants for common usage is still valid. How do you think? @Wadeewee

Reply:
Agree @hoangnguyen92dn , when using common constant, we usually need to check its value (unless we're working on it long enough to remember 😄 ). So in this case, if we cannot find a well common name, then we can use the number directly.
When defining `space10 = 10.0`, we could not change its value anymore. In the future, if we need to change the size of the component using it, we have to do it in appropriate places too.

Reply:
> When defining space10 = 10.0, we could not change its value anymore. In the future, if we need to change the size of the component using it, we have to do it in appropriate places too.

Regarding this case, I think changing the reference in widgets using the constants could be easier for the developers and also code reviewers to be aware of the scope of change, rather than we change the value in constants definition and we might need to check the whole project which widgets are using that constant 😄

Btw, it's just the way we name the constants and manage the dimensions, with the IC's scope, I believe it's still acceptable.

Reply:
@hoangnguyen92dn Can we remove these dimens:

> const double space12 = 12.0;
> const double space14 = 14.0;

I didn't see where we use it 🙏

Reply:
@Wadeewee Yeah. Actually, I created those for later usage 😄 We don't use it currently indeed. lets me remove.